### ISSUE TITLE
0.9.7 Balance Issues: Are these as intended?

### ISSUE THREAD

**Issue by marphod** [endless-sky/endless-sky#2852]
Playing through the new content, I've noticed a few balance issues along the way.

- Wanderer Transport Missions (& other missions involving transport) seem to have a ridiculously high payout.  

Not that it is necessary excessive per sentient transported, but they are really large groups over a very short ranges, and often you will be offered multiple missions on the same path (since there are only so many options in the territory that small).  Comparable Bulk Cargo missions would need to be about an order of magnitude higher (300-2000 units rather than 30-200) to have similar payouts, but that just doesn't happen.  (For instance, I've got 4 cargo missions to the same planet that total approx 275 tonnes of goods and pays approx 230k credits; which is way better than average.  I've got got 4 transport missions for 185 wanderers to an equally distant system for 1.2m credits; and that's on the low side of average.)  I've even seen transport missions with a 5m+ payout.

This may be a side order of 'Transport missions scale better with character power than cargo missions'.

- The Remnant Crystal Capacitor seems very powerful

And/or the Coalition outfits are terrible. 

The Crystal Capacitor has a capacity of 283 units of energy per ton, and is under 20 tons in size.  The only other small footprint battery with similar capacity is the Large Battery Module at 250 units of energy per ton -- which was one of the only  useful items the coalition had.  (The only other batteries that compare are the Quarg Nanotech, which is Tier 4-ish, and possibly the Korath Large System Core and/or the Wanderer Sun generators; and those are much larger footprint at at least 50 tonnes and that depends on how you calculate the value of the multi-function outfits).

- Outfit Expansion Nerf may not be enough of a Nerf -- 

It doesn't hurt large-battery/small-generator builds

There are issues with that style of ship, but a Bactrian with a Large Biochem or Red Sun, 2-3 System Cores (Large), some batteries (Nanotech are best, but Crystal Capacitors or Large Battery Modules will do) and a defensive flagship load-out is a working build , doesn't care about the lack of cooling, and is fine in all but the longest firefight.  Throw a cloaking device in with enough ramscoops and it is even better.  You can do the same with most Missile-boat builds.

(I may find more)

**Comment by Bladewood** [comment]
Wanderer missions are intentional.

The Crystal Capacitor is much more space efficient than the Large Battery Module, which is itself much more efficient than pretty much every other dedicated battery outfit in the game (ignoring nanotech batteries). I'm certain this is intentional.

If it "doesn't hurt large-battery/small-generator builds", then that's great.

**Comment by Amazinite** [comment]
>Outfit Expansion Nerf may not be enough of a Nerf --
It doesn't hurt large-battery/small-generator builds

That's pretty much the intention of the outfits expansion. And you might be able to get around the lower cooling by using lots of batteries instead of large generators, but that leads to a ship which 1) can't sustain itself over an indefinite amount of time and 2) such builds also require weaker weapons as stronger weapons run hotter. A ship using no outfits expansions is probably going to win in the long run against a ship using outfits expansions + a large battery/small generator build to get around the decreased cooling just because of the fact that the ship without outfits expansions can sustain itself for long and is not hampered by the lack of cooling and therefore can use stronger weapons.

**Comment by MessyMix** [comment]
With regards to the issue of Wanderer space being too profitable: as @Amazinite said, more advanced species = higher payout.
[#2800](https://github.com/endless-sky/endless-sky/issues/2800#issuecomment-318236607) should also address some part of this issue.

And as for the Outfits Expansion: The point of the nerf is to stop the Bactrian from being insanely overpowered as a wheeling ship of death. Now, you are relegated to being a "defensive flagship", which is exactly the point.

My old Bactrian, converting only 200 cargo space, can run 2 skylances, 2 blue suns, and could easily defeat any <T2 ship. Good luck doing that with 200 cargo space worth of outfit expansions.
End point: outfits expansions seem to be working. :)

**Comment by ghoulavenger** [comment]
The bactrian OP? Really? Other than having a large crew capacity (so it is good for capping ships, no other ship compares really) it's strength is it's versatility, not it's construction. Shield beetles and Kestrels (I loved the Kestrel until the game pretty much forced me into Hurricanes -- them drones are dangerous) are significantly better warships, and don't require jump drives to get. Pretty much the first Wanderer warship you can buy is better too. If I was to really complain about the bactrian I would rather complain that there is no significantly better flagship for capturing purposes, only better for combat purposes, which means you either fly a ship that can capture other ships effectively, or one that won't die. I understand the reasoning behind not making stronger ships for capping though, I'm pretty sure the korath worldships (despite being significantly worse than the bactrian, unless perhaps you had androids modded in, but even then it's not as flexible as a ship) and quarg ships are not intended to be capturable. You can fix both of those problems though by simply disabling their ability to be boarded. It's not like you aren't doing that already, some ships will self destruct frequently pretty much accomplishing the same thing, and the drak ships can't be disabled.

**Comment by Amazinite** [comment]
>The [B]actrian OP? Really? 

During the Age of Mass Expansions, as I have just dubbed it such, the Bactrian could easily be converted into a warship on-par with or better than the Kestrel or Shield Beetle, while also sustaining a large amount of cargo for a warship (which I still despise, plz nerf to the 250-350 range) and its unmatched boarding ability (which I would nerf for the sake of adding room for more boarding ships between the Bactrian and World-Ship without making any new boarding ships immediately allow you to cap a World-Ship, as the Bactrian is currently just on the edge of being able to cap a WS, but only because of its combination of high base bunks and high cargo that allows you to increase your number of bunks; a bunk maxed Bactrian can have 465 crew). 

And as Messy has just said:
>[The] old Bactrian, converting only 200 cargo space, [could] run 2 skylances, 2 blue suns, and could easily defeat any <T2 ship.

While also tussling pretty easily with lower-end T2 ships. It was a nightmare and I'm glad outfits expansions are a thing, but I'll still sit here and complain about the balance of the Bactrian, because outfits expansions don't change the fact that it's imbalanced. :)

**Comment by marphod** [comment]
> What's the problem in that? Some outfits are going to be more powerful than others. And it's only 13% more efficient than the Large Battery Module, so I wouldn't exactly call the Coalition outfits "terrible."

The problem is there is now no reason to use any coalition tech -- they are inferior to any other T2 culture.  I would expect each culture to have strengths and weaknesses, but the coalition lacks any real best-in-class options.


Of the T2 cultures: 
The Korath have best in class shield/hull/batteries (Large System Core is amazingly efficient, the medium is very strong), the most powerful engines-per-mass, the best in game passive cooling, and some unique weapons.  There generators run hot but are nothing to sneeze at either.
The Wanderers have the best-in-class ship frames, the best-in-class generators+batteries, 2nd most powerful turret weapon  in the game, and the best available ramscoop.
The Remnant have the best stand-alone battery, best-in-class low heat engines, and a few neat items i want to check out.
The Coalition have ... good personality?  a large territory? lots of mediocre frames? And a hull repair module if you don't have the mass for a medium or large System core.

In many ways, the Coalition feels even lower tier than the Hai.  And yet, they overthrew the Quarg.

--

Lets see.  A low-engine Bactrian:
- 3 System Core (Large)
- 2 Skylances
- 2 Quarg anti missiles
- 1 Nanotech battery (all of  these from a single quarg ship)
- 1 Red Sun
- 14 Outfit Expansions
- 12 large battery modules 
- Bondir thruster & steering

That's nearly 30 seconds of sustained full firepower with intermittent throttle and running near but bellow heat capacity.  Swap in some Remnant engines and crystal capacitors-- or nanotech batteries --  and you'll be able to run longer and cooler.  With over 1000shield regen a second and nearly 150 hull repair.

That's not the 4 skylance version i was running in my first playthrough, but it still can take on nearly any t2 ship and win.

**Comment by MessyMix** [comment]
@marphod okay, now pit that Bactrian against any ship running any ion or heat damage (i.e. wanderers), and you're screwed.

Let's not forget that the Coalition is a roughly tier 1.5 civilization, which makes them by default inferior to a tier 2 race. Their freighters aren't competitive with Tier 2 freighters (look at the Deep River, then look at the Arach Hulk). Only the Heliarch are T2+ (roughly 2.5, they are above the Wanderers).

And, their cooling module is exceptional. That's what I use Coalition tech for. And, whenever I don't have the space to have a Crystal Capacitor, I use their batteries. And when I need extra kick in my engines for minimal outfit space, I use theirs.

So no, Coalition tech isn't 100% obsolete, and they are not a fully fledged Tier 2 race in terms of ship frames.

**Comment by ghoulavenger** [comment]
With it's base stats so much lower, and its drag so much higher, I couldn't put the Bactrian on the Kestrel's level, well, pretty much ever. With the conversion of outfit space you could argue the bactrian could support some mildly larger weapons than the Kestrel, but at the cost of speed and survivability. And I don't like throwing away money on fighters (I might run boxwings for pseudo cargo space, or korath chasers for show but never deploy them), although if you run a fleet of fighters you can pretty much kill everything (it's just expensive, at that point we might point out that sparrows are probably the most deadly ships in the game for the cost, you'll just lose a lot in big battles).

Are there broken situations? Yes. The extra cargo capacity turned into outfit space can really produce some broken shield generator setups, and quarg skylances are pretty broken on their own. You can also make a few sacrifices if you wanted to run cloaking indefinitely that you might not on other ship types.

Honestly though I thought the outfits expansion bit was not directly targetted at Bactrians, although they are certainly one of the worst ones hit by it. I thought they were so you could bump up the cargo space on Deep Rivers. While I wouldn't consider them the best flagship possibility by a long shot (terrible engine choices with no turrets so basically sitting ducks on their own), with that extra outfit space you could make them decent crew carriers with the large cargo space increase. But with the outfits expansion you might not want to do that, especially considering you won't get a stronger capturing ship than the Bactrian (though it would have been more survivable).

**Comment by jafdy** [comment]
The move to outfit expansions from mass expansions was to allow large cargo ships without them becoming overly good warships. Yes, it was conceived as a way to fix the Bactrian's problems but it has also allowed the cargo capacity of other ships to be increased.

**Comment by KiLEdEnNis** [comment]
about you calling the Coalition shit
the civilians are only around T1,5, below T2, while the Korath Automatons and end-game Wanderers are easily over T2, around T2,5
the actual Heliarch can be compared to them; their warships are extremely resilient (the Punisher is about as good a flagship as the Hurricane) / their Finisher Torpedoes are extremely strong / the ion rain gun is very small utility / the Attractor/Repulsor combo can be very useful / their active cooling is VERY good / you can easily nearly fill your entire engine space with their engines
the shield/hull regen outfits are underwhelming but also cheap, so calling them bad because they excel at nothing seems very wrong to me
(and correct me if I'm wrong about something here @anyone)

**Comment by MessyMix** [comment]
Almost right, but not quite :P. the Korath Automata and Wanderers are pretty much the benchmark for T2 iirc, because if we follow the admittedly dated rule of TX Heavy Warship = T(X+1) Light Warship, we find that in terms of survivability Wanderers and Automata stack up quite nicely to one flat tier above humans.

The Heliarch are stronger yet, the Punisher clocking in at over 100,000 shields, making them more powerful than either the Korath or the Wanderers. And yes, their finishers outclass anything.

While the Heliarch reactors may seem underwhelming—which, statistically speaking, they are in terms of mass efficiency—they are still solid choices, and better than Hai reactors. I'm surprised that for 180 outfit space, you can stack 2 large reactors and have over 2.6k energy generation—which is comparable to a Triple Plasma Core + a Large Heat Shunt.

**Comment by Amazinite** [comment]
>The problem is there is now no reason to use any coalition tech -- they are inferior to any other T2 culture.

*Cracks knuckles*
As said, the Coalition are T1.5 and therefore will be inferior in many ways to higher species, but lets go through this.

I'm going to lay out the best outfit from each class (weapons excluded, and ignoring T3+ tech) just so we can see how "terrible" the Coalition (or any other faction) are.

* **Best thrusters**: Pug thrusters (Pug Lohmar), followed by Wanderer thrusters (Type 4 Radiant), followed by Hai thrusters ("Bufaer" Atomic). But that doesn't mean you're always going to want to choose Pug or Wanderer thrusters over Hai thrusters. The reason I order them like this is because Pug thrusters top the charts when it comes to both thrust/energy and thrust/heat, while also giving substaintial thrust/ton. But Pug thrusters aren't exactly very obtainable, so Wanderer thrusters, being more powerful in terms of thrust/ton and only slightly less thrust/heat efficient but more thrust/energy efficient, come next. Hai thrusters though combine both massive power and great efficiency, so I would consider them better as combat thrusters than the other two.
  * Sidenote: Korath thrusters do not win in this territory because of their TERRIBLE thrust/heat. Aside from Coalition thrusters and afterburners, all 5 Korath thrusters have the worst thrust/heat in the game. Their thrust/energy values are also poor, with Stellar Thrusters coming out at 26th of 45 thrusters in terms of thrust/energy, and Asteroid Thrusters all the way down at 40th of 45.
* **Best steering**: "Bufaer" Atomic Steering. Same reasoning as for the best thruster, only the Hai huge steering wins out because the Pug and Wanderer large steering outfits don't compete as well as their thruster counterparts.
* **Best reactor**: *Technically* the best power generation to have would be the Large Collector Module. 7.07 energy/ton (which would increase closer to the system center) and only 0.45 heat/energy. Solar collection aside though, the Blue Sun Reactor comes out on top as the best reactor. It provides 12.36 energy/ton at a measly 1.25 heat/energy. Only five reactors produce more energy per ton that the Blue Sun Reactor, but all produce far more heat (see below). Ignoring Wanderer reactors though, I would consider the Large Reactor Module to be the next best power generator, as it has the 3rd highest energy/ton but is not burdened by the massive heat/energy that the Double or Triple Plasma Core have.
  * Small Reactor Module: 12.57 energy/ton at 2.16 heat/energy
  * Plasma Core: 13.22 energy/ton at 3.03 heat/energy
  * Large Reactor Module: 13.65 energy/ton at 2.02 heat/energy
  * Double Plasma Core: 14.19 energy/ton at 3.27 heat/energy
  * Triple Plasma Core: 15.48 energy/ton at 3.53 heat/energy
* **Best cooling**: Cooling Module, no contest. It provides 252 cooling/ton, which is 68% better than the next best cooling outfit, which is the Large Heat Shunt at 150 cooling/ton. The 84 energy/sec that the Cooling Module uses at most (not all the time) is well worth the massive increase in cooling. 
* **Best battery**: The Crystal Capacitor at 283.33 energy/ton for 12 tons. But, I'd say that the Small Battery Module (not the large) is a close second due to its modularity. It provides 225 energy/ton at only 4 tons, a third that of the Crytsal Capacitor. So if you're trying to squeeze some more energy in or just don't have enough remaining space for a Crystal Capacitor to act as your battery, then the Small Battery Module is the way to go.
* **Best shield generator**: Either Dark Storm Shielding or the Large System Core depending on what you're look for. Dark Storm Shielding provides 5.12 shield per ton at 2.00 energy per shield. The Large System Core provides 3.96 shield/ton at 1.47 energy/shield. That means that Dark Storm Shielding is 29.3% better in terms of shield/ton but 36% worse in terms of energy/shield. In conclusion, if you have enough energy to spare then you'll want Dark Storm Shielding, but otherwise get a Large System Core. At as we all know, the Large System Core does more than just generator shields, so some may choose it over Dark Storm Shielding in all cases because of that.
* **Best hull repair**: The Large Repair Module, whos hull repair is better than Korath System Cores. System Cores require 1 energy per hull and produce 1.5 heat per hull repaired. By comparison, Repair Modules require the same 1 energy per hull but produce only 0.83 and 0.88 heat for the small and large modules respectively. They also have more hull repair per ton. 
  * Large System Core: 0.53 hull repair/ton
  * Large Repair Module: 1.63 hull repair/ton (207.5% increase)
* **Best ramscoop**: The Wanderers at 0.57 ramscoop per mass, 30.3% better than the Catalytic Ramscoop, which is the next best.

So in the end, here's how it pans out:
* **Humans**: [This](https://www.youtube.com/watch?v=M5QGkOGZubQ)
* **Hai**: Best steering and 3rd best thrusters.
* **Pug**: Best thrusters
* **Coalition/Heliarch**: Best hull repair, second best batteries, best cooling, and next best reactors.
* **Wanderers**: Best ramscoop, best reactors, best shield generator given you have energy to spare, and the 2nd best thrusters (and although not mentioned above, the 2nd best steering too).
* **Korath**: Best static cooling (Large Heat Shunt) and the best shield generator if you can't support Wanderer shield generators.
* **Remnant**: Best batteries... that's it. (Also the 3rd best steering, if it matters.)

If anything, I'd say the Remnant are looking pretty poor right now.

**Comment by Lineth** [comment]
I'd say that Heliarch reactors are better than Korath reactors, personally. Of course, Wanderer reactors are the best.

**Comment by marphod** [comment]
You really can't consider the multi-function equipment in isolation like that.  Yes, the system core isn't as efficient space wise as the large repair module, but that's not all that it does.

You need 1 and a third Large Repair Modules (ea @ 21t, 34energy/repair, 30 heat) plus 1 1/2 Dark Storms (ea @ 41t, 420e, 210shield)  and 1 2/3Crystal Capacitors (ea @ 3400 e, 12t) (that is, a fraken-outfit that is approx 108t, ~45hull repair at ~45energy, 40 heat/repair, 375 shield at 750 energy, 5650 energy capacity) for the functionality of 1 Large system core (91 t, 48 hull repair @ 48 energy, 72 heat; 360 shield at 528 energy, 5600 energy storage).  If you have the space, why would you get anything else?

Similarly, with reactors, nothing even comes close to the Blue Sun.  A triple core (2616 energy at 169t) compares amazingly well with the blue sun (1878 @ 152) until you consider the energy storage.   A theoretical scaled-down Triple Core would only weight 121 t to provide the same energy, but you would need more than 2 1/2 Capacitors to add the energy storage (30t, 8100e) at which point, the triple core is directly comparable with the blue sun; and then there is the triple core's heat.  Large Reactor Modules? You'd need 137t worth of them, plus the batteries, meaning you are spending 15t more for less storage.

---

I will admit i didn't take the Heliarch equipment into consideration; but as there is no in-game way to get a Heliarch license yet, and they are, at this point, a friendly culture without cause for hostilities, I've not explored their mil-tech.

**Comment by MessyMix** [comment]
Marphod, your initial argument was that Coalition tech was useless, as far as I am aware. "The problem is there is now no reason to use any coalition tech."

It's uses are: Cooling Module, Batteries (for extra kick, where Crystal Capacitors are too big to fit), and hull repair, if you're not using System Cores.
Heliarch tech is a whole new story—as you seem to know. Finisher torps are devastating (it's the real highlight), and their hulls are beyond T2.

If we're talking about statistics, as you say, we can't consider multi-function equipment in isolation. This statement is a double-edged sword: a) mono-function equipment can be compared laterally, and b) we can't consider multi-function tech without considering complementary tech.

[by the way, this bit isn't directed at you alone, it's more just.. food for discussion :)]

Let's start with the second point: you can't consider multi-function tech without considering its complementary technologies.

If we look at the Blue Sun reactor, it's easy to say that it's the best reactor, since it is multifunction, and it has energy storage. It makes the TPC pale in comparison, with it's high heat generation, and 0 storage. But we have to remember that the TPC was designed to be paired with System Cores, which fill in the role of what the TPC lacks in comparison to the Blue Sun.

So, our complementary technologies:
Blue Sun <--> Dark Storm :: TPC <--> System Cores
Essentially:
Multipurpose <--> Mass Efficient :: Mass Efficient <--> Multipurpose

This makes it very hard to say that one build is strictly better than the other—and even so, it's marginal at best. Yes, you could say that a Blue Sun <--> System Core build would wreck everything, but then you just end up with a build that has way too much energy capacity, and is very mass inefficient.

tl;dr: System Cores are the Blue Suns of shielding [multipurpose], and Dark Storms are the TPCs of shielding [mass-efficient], so when these two technologies are attached, you can't say the Blue Sun is strictly the best Generator. 

Standalone, yes, Blue Sun all the way—but it depends on the rest of your build. If you have System Cores, the TPC isn't a bad choice, since you already have the energy storage, and then it's *redundant* to use a Blue Sun, because you should go for a more mass-efficient generator, especially if you are using a power hungry build. See what I'm saying?

Now, Part A: single-function outfits can be compared laterally.
This means that we can compare the TPC directly to 2 Large Reactor Modules. This would work best with System Cores, since they complement the single-function reactors.

TPC: 169 outfit space (193 with LHS), 2616 energy generation, 9240 heat generation (6000 with LHS)
2 Large Reactors: 196 outfit space, 2676 energy generation, 5400 heat generation.

In short, the Large Reactor Modules are actually more mass efficient than the TPC, making them a better choice for System Core builds. :)

/endrant

**Comment by Amazinite** [comment]
>You really can't consider the multi-function equipment in isolation like that.

Whether you can or you can't doesn't really relate to the fact that you said Coalition outfits are useless, despite the fact that they have the best cooling, best dedicated hull repair (which you can buy and doesn't need plundered), second best dedicated batteries (and a much more modular one as well), and what amounts to the second best reactors (whether looking at Wanderer reactors' energy capacity or not).

And dedicated outfits always have the edge over hybrid outfits in that they're smaller. If I need over 5k energy capacity and I'm limited on space, I'll be installing two Crystal Capacitors for 24 tons, not a 91 ton System Core. That's why I will look at the stats of an outfit in isolation but still mention their hybrid nature (as I did when comparing the Large System Core and Dark Storm Shielding), as I want to know what's going to be the best outfit for the singular purpose I'm looking for when I'm limited on space.

>You need 1 and a third Large Repair Modules...

Large System Cores actually store 5,200 energy, not 5,600 energy. Also, do it like this:

* 5200 energy capacity of the LSC/283.333 energy per ton of the Crystal Cap = 18.35 tons worth of Crystal Cap
* 48 hull repair of the LCS/1.63 hull per ton of the LRM = 29.45 tons worth of Large Repair Module
  * 48 hull energy (0% change)
  * 42 hull heat (41.67% decrease compared to LCS)
* 360 shield gen of the LCS/5.12 shield per ton of the DSS = 70.31 tons worth of Dark Storm Shielding
  * 720 shield energy (36.36% increase compared to LCS)

Overall that's about 118.11 tons of dedicated outfits (29.8% increase) to match the Large System Core in energy capacity, hull repair, and shield generation, while being 41.67% more hull heat efficient and 36.36% less shield energy efficient.

>Similarly, with reactors, nothing even comes close to the Blue Sun.

Which I already agree with, but now I feel like doing the exact calculations:

* 8900/283.333 = 31.41 tons worth of Crystal Capacitor
* 1878/13.65 = 137.58 tons worth of Large Reactor Module
  * 3793.56 heat (62.1% increase)
  * 168.99 tons of Large Reactor Module and Crystal Capacitor (11.2% increase) to match the energy capacity and energy generation of the Blue Sun Reactor at 62.1% more heat
* 1878/15.48 = 121.32 tons worth of Triple Plasma Core
  * 6629.34 heat (183.3% increase)
  * 152.73 tons of Triple Plasma Core and Crystal Capacitor (0.48% increase) to match the energy capacity and energy generation of the Blue Sun Reactor at 183.3% more heat (have fun with that).

>A triple core (2616 energy at 169t) compares amazingly well with the blue sun (1878 @ 152)

You always either forget about or completely disregard heat, don't you?
 * Blue Sun Reactor: 12.36 energy/ton at 1.25 heat/energy
 * Triple Plasma Core: 15.48 energy/ton at 3.53 heat/energy
These don't compare well AT ALL. Yes, the Triple Plasma Core is about 25% more energy efficient than the Blue Sun Reactor, but at a painful cost of 183% more heat! That does not look well for the Triple Plasma Core at all, whether taking into account the energy capacity of the Blue Sun Reactor or not. And after adding cooling to account for the massive increase in heat and taking the weight of the cooling into account, the Triple Plasma Core becomes worse than the Blue Sun Reactor.

**Comment by SirStigII** [comment]
I'd just like to add that the Remnant PDT is by far the [best ](https://docs.google.com/spreadsheets/d/1Oeuc1FDi0zV0dl2pZbktyw0LbJnMYRhV7hNKUW5lD2Y/edit?usp=sharing)single anti-missile turret if you have the space. The warder is better vs meteors, but the PDT is so much better at everything else it's almost funny.

**Comment by Bladewood** [comment]
http://steamcommunity.com/app/404410/discussions/0/1458455461496380156/#c1458455461496439996 ~~I have no life.~~

**Comment by ghoulavenger** [comment]
I always consider an outfit at its worst (because I build my fleet to be flexible), and I find it hard to justify the active cooling of the Coalition over the Large Heat shunts passive cooling. It takes roughly 5 coalition coolers to match 2 shunts (5.1428 or something, rounded down since you can't have partial outfits, and 6 is probably far more than you need if 2 shunts is acceptable for your cooling needs, for reference I use 2 shunts). That's a possible 420 energy/sec to save 23 outfit space. Maybe if you had a lot of extra energy to burn anyway they would be a viable option since 23 outfit space is potentially an extra outfit, but 23 outfit space isn't enough to install a reactor to cover that energy cost (not to mention the extra heat it would generate so you would probably have to cover it with battery power to maintain efficiency anyway, which would make them far less space efficient compared to the shunts). So I wouldn't want to rely on them as primary cooling for the heaviest of possible outfits, which is where they would be the strongest.

The major potential use I see of their cooling module is emergency cooling. Where that cooler could be the differance between overheating and not, like against wanderers.

**Comment by gunqqer** [comment]
Coalition cooling mods are really good. My ships don't require massive cooling (But still need some). All I need is 2-5 cooling mods and I am set for cooling. Also energy isn't a real problem becuase they need much less than everything else and a nanotech batt+3 blue suns means that adding the cooling mods instead of heat shunts is half a second less fire time (By which everything would be *very* dead). I don't consider an outfit at its worst, I consider it at its best and what I can do with it then. Play to its strengths not harp over its weaknesses (But take them into account).  
Unless I was using Korath weapons and reactors I would have to use heat shunts because of the limitations of the inefficency of korath tech. But I don't.

**Comment by ghoulavenger** [comment]
I don't use anything Korath other than the shunts. But you made my point for me already. You've dedicated a third blue sun that I did not need to. With two blue suns, I can run a full armament of dual sunbeam turrets (4 on my hurricanes because I want at least one AM), and with two heat shunts completely manage my heat requirements, including running Hai atomic engines almost indefinitely (it's actually about 30 seconds if I'm running my engines, guns, shield generators and hull repairers at the same time, but this is almost never true -- since beam weapons deactivate automatically, and engines only provide thrust when you need to accelerate -- which gives ample time for the batteries to recharge). Remember, I said considered it this way because I wanted flexibility (which requires making sacrifices to be able to adapt to the situation). By most peoples standards, they would probably find that my strength is in my fleet, not in any individual ship.

**Comment by marphod** [comment]
I more or less ignore heat for one basic reason: I've never found it to be the limiting factor in a build, except for (now) high outfit expansion builds.

Weapon space and engine space are limiting factors.  Occasionally i find I dont have enough or the right  hardpoints. I have to compromise on weapons or engines to fit a fuel pod, or scanners on my flagship.  Rarely, I have to choose between  hyperdrive and a jump drive, rather than both. But I rarely find that I need more than a single large shunt and maybe an extra small or two, and plan that from the start.

The only time I have heat issues are if I strip the shunts out of a Kor Sestor fighter/drone and then succeed in capturing it.


If there were sizeable hostile NPC fleets that ran Sunbeams or Plasma, I might have issues, but there aren't.  The rare NPC that does use heat weapons usually falls to my fleet barrage long before they're a threat.


----

I find a Blue Sun and a Large system core are the basis  for nearly every capital ship build; smaller warships might require a medium core or white or yellow sun instead, but the energy capacity is great and nearly perfect.  It means I have  enough storage to run at a reasonable power deficit without any loss of effectiveness for the length all but the longest fleet v fleet conflict.

In the rare build that needs two reactors, the extra capacity is also necessary, so still a system core.

**Comment by MessyMix** [comment]
So. Have we cleared up all your gripes with 0.9.7 balance? Seems like we addressed most of your initial points already.

1. Wanderer Transport Jobs (and T2 jobs in general) are intended to be higher payroll.
2. Coalition tech has its uses. Cooling, reactors, hull repair, and Heliarch military tech. (even if you "ignore heat", you still need to deal with the 6k heat coming from your Blue Suns and Systems Core, and for mass efficiency, Coalition cooling is the way to go.) Again, as I said above, when I use Systems Cores, TPC's work better than Blue Suns, and Coalition Reactors are actually more efficient than TPC's.
3. Outfits Expansion was nerfed significantly. You can tell it's succeeded, because "now high outfits expansion builds" need you to deal with heat, as you said yourself. So it's fulfilled its intended purpose. :)

**Comment by marphod** [comment]
Actually, 1 got left in the dust.

 No one addressed my point that the Transport missions (i.e. missions that take sentients, with or without some cargo) scale with the player/culture level, whereas the pure Cargo missions don't.  It is particularly noticeable with the Wanderers, as they had just gotten custom missions.

If the payout for Transport, and the headcount for those missions are as intended, then the cargo missions in Wanderer space (and possibly any T2 space) should be an order of magnitude larger to have comparable pay ranges.
--

6k heat? System Cores + Blue Sun is 3k, not 6.  I rarely run multi-reactor systems.  That works fine with a single large heat shunt.  I'm more likely to add a system core than a second reactor and 1 shunt is more than enough cooling on 2/3rds of my current fleet, as long as I stick to 5 outfit expansions or fewer (Not that I use that many; its rare I need more than 2 i any).  I was trying a dual reactor on my other third, so I added a second shunt for some extra cooling.  Turns out, the dual cores work better.

The high count outfit expansion builds means a smaller reactor and more system cores.  I don't run them, as the Hurricane doesn't need it, and I've phased out my  Marauder Leviathans and Kestrels that did.  Future playthoughs, I'll probably go directly from a Firebird Fleet to Shield Beetles (captured from the unfettered) and skip the Kestrel and Leviathans entirely (and other human heavy warships other than a bactrian flagship -- notbecause of the nerf, but because it is just a pain to do the transitions, so dropping 2 out of my cycle makes life better)).

**Comment by endless-sky** [comment]
The point of Remnant tech is that it's a mix of really powerful (found or stolen) things, slightly-improved human tech, and just plain weird (also found or stolen) things.

I've balanced the Crystal Capacitors a bit in 1ec40a8 by making them consume a small amount of energy and generate some heat, while keeping their storage per ton unusually high. To compensate, I also improved their generators a bit, although their size efficiency is still well below even human nuclear tech.

**Comment by KyroFirehammer** [comment]
I could ne wrong on this but with the new nerf on outfit expansion does it mess with marquees because they used mass and it was ok but the usually run hot in my experience so their might be a problem w/ them

**Comment by SirStigII** [comment]
Marquees?

**Comment by gunqqer** [comment]
@KyroFirehammer when ever you post something you always have atleast one typo in it, your grammar isn't very good, and instead of editing posts you reply to them. Here on github we like to to understandable, so please take more time with what you post or go to the steam forums
